README.md

```diff
@@ -13,38 +13,42 @@ RESCRIPt is a python 3 package to support a variety of operations for managing a
 
 RESCRIPt will be installable as conda package in the near future. In the meantime, we provide two routes for source installation: a minimal RESCRIPt environment, or within an existing QIIME 2 environment:
 
-### Option 1: Minimal RESCRIPt environment:
-First create a conda environment and install relevant dependencies:
+### Option 1: Installed as part of a QIIME 2 distribution.
+
+For QIIME 2 versions `2023.9` and later, RESCRIPt is installed as part of the QIIME 2 "Shotgun Distribution", as outlined in the [QIIME 2 Documentation](https://docs.qiime2.org/). RESCRIPt is available as part of the "Amplicon Distrubtion" for QIIME 2 releases `2024.2` and later.
+
+### Option 2: Minimal RESCRIPt environment 
+
+To make use of the latest "minimal" RESCRIPt release, some components of QIIME 2 (releases `2023.9` and later) are required:
+
+First create a conda environment and install relevant dependencies using either `conda` or `mamba`. 
+*Note: update `{ENV_VERSION}` in the commands below to match the QIIME 2 release.*
 
 ```
 conda create -y -n rescript
 conda activate rescript
 conda install \
-  -c conda-forge -c bioconda -c qiime2 -c https://packages.qiime2.org/qiime2/2023.5/tested/ -c defaults \
-  qiime2 q2cli q2templates q2-types q2-longitudinal q2-feature-classifier 'q2-types-genomics>2023.2' \
+  -c https://packages.qiime2.org/qiime2/{ENV_VERSION}/shotgun/passed/ \
+  -c https://packages.qiime2.org/qiime2/{ENV_VERSION}/amplicon/passed/ \
+  -c conda-forge -c bioconda -c qiime2 -c defaults \
+  qiime2 q2cli q2templates q2-types q2-types-genomics q2-longitudinal q2-feature-classifier \
   "pandas>=0.25.3" xmltodict ncbi-datasets-pylib
 ```
+
 Install source:
 
 ```
 pip install git+https://github.com/bokulich-lab/RESCRIPt.git
```

Comment:
RESCRIPt should now be conda installable via the same channels above, so this should not be necessary.

Reply:
Doh! 🤦